$NetBSD: patch-data_meson.build,v 1.1 2022/03/10 21:25:30 nia Exp $

Remove incorrect positional arguments, previously these would have
been silently ignored but now they cause build failures.

--- data/meson.build.orig	2017-10-01 14:14:30.000000000 +0000
+++ data/meson.build
@@ -1,5 +1,4 @@
-i18n.merge_file ('desktop',
-                 input: 'org.gnome.Dictionary.desktop.in',
+i18n.merge_file (input: 'org.gnome.Dictionary.desktop.in',
                  output: 'org.gnome.Dictionary.desktop',
                  install: true,
                  install_dir: join_paths(gdict_datadir, 'applications'),
@@ -14,8 +13,7 @@ if desktop_file_validate.found()
        workdir: meson.current_build_dir())
 endif
 
-i18n.merge_file ('desktop',
-                 input: 'org.gnome.Dictionary.appdata.xml.in',
+i18n.merge_file (input: 'org.gnome.Dictionary.appdata.xml.in',
                  output: 'org.gnome.Dictionary.appdata.xml',
                  install: true,
                  install_dir: join_paths(gdict_datadir, 'appdata'),
@@ -48,8 +46,7 @@ sources = [
 ]
 
 foreach s: sources
-  i18n.merge_file('desktop',
-                  input: '@0@.in'.format(s),
+  i18n.merge_file(input: '@0@.in'.format(s),
                   output: s,
                   install: true,
                   install_dir: join_paths(gdict_datadir, 'gdict-1.0', 'sources'),
